modules/aws-sso/main.tf

```diff
@@ -19,24 +19,12 @@ module "permission_sets" {
 
 module "sso_account_assignments" {
   source  = "cloudposse/sso/aws//modules/account-assignments"
-  version = "1.1.0"
+  version = "1.1.1"
 
   account_assignments = local.account_assignments
   context             = module.this.context
 }
 
-module "sso_account_assignments_root" {
-  source  = "cloudposse/sso/aws//modules/account-assignments"
-  version = "1.0.0"
-
-  providers = {
-    aws = aws.root
-  }
-
-  account_assignments = local.account_assignments_root
-  context             = module.this.context
-}
-
 locals {
   enabled = module.this.enabled
```

Comment:
we'll also want to remove the locals starting with `local.account_assignments_root` and related locals

Reply:
My mistake. We need the `root` provider and locals, and I have restored them.